pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -301,46 +369,50 @@ func TestDesiredRouterDeployment(t *testing.T) {
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_CAPTURE_HTTP_COOKIE", false, "")
-	checkDeploymentHasEnvVar(t, deployment, RouterDontLogNull, false, "")
-
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_BUF_SIZE", false, "")
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_MAX_REWRITE_SIZE", false, "")
-
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_CIPHERS", true, "foo:bar:baz")
-	checkDeploymentDoesNotHaveEnvVar(t, deployment, "ROUTER_CIPHERSUITES")
-
-	checkDeploymentHasEnvVar(t, deployment, "SSL_MIN_VERSION", true, "TLSv1.1")
-
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_IP_V4_V6_MODE", false, "")
-
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_UNIQUE_ID_HEADER_NAME", false, "")
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_UNIQUE_ID_FORMAT", false, "")
-
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_H1_CASE_ADJUST", false, "")
-
-	checkDeploymentHasEnvVar(t, deployment, RouterHAProxyThreadsEnvName, true, strconv.Itoa(RouterHAProxyThreadsDefaultValue))
-
-	checkDeploymentHasEnvVar(t, deployment, RouterHTTPIgnoreProbes, false, "")
-
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_DEFAULT_CLIENT_TIMEOUT", false, "")
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_CLIENT_FIN_TIMEOUT", false, "")
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_DEFAULT_SERVER_TIMEOUT", false, "")
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_DEFAULT_SERVER_FIN_TIMEOUT", false, "")
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_DEFAULT_TUNNEL_TIMEOUT", false, "")
-	checkDeploymentHasEnvVar(t, deployment, "ROUTER_INSPECT_DELAY", false, "")
 
-	checkDeploymentHasEnvVar(t, deployment, RouterEnableCompression, false, "")
-	checkDeploymentHasEnvVar(t, deployment, RouterCompressionMIMETypes, false, "")
+	tests := []envData{
+		{"ROUTER_LOG_FACILITY", false, ""},
+		{"ROUTER_LOG_MAX_LENGTH", false, ""},
+		{"ROUTER_LOG_LEVEL", false, ""},
+		{"ROUTER_SYSLOG_ADDRESS", false, ""},
+		{"ROUTER_SYSLOG_FORMAT", false, ""},
+		{"ROUTER_CAPTURE_HTTP_REQUEST_HEADERS", false, ""},
+		{"ROUTER_CAPTURE_HTTP_RESPONSE_HEADERS", false, ""},
+		{"ROUTER_CAPTURE_HTTP_COOKIE", false, ""},
+		{RouterDontLogNull, false, ""},
+		{"ROUTER_BUF_SIZE", false, ""},
+		{"ROUTER_MAX_REWRITE_SIZE", false, ""},
+		{"ROUTER_CIPHERS", true, "foo:bar:baz"},
+		{"ROUTER_CIPHERSUITES", false, ""},
+		{"SSL_MIN_VERSION", true, "TLSv1.1"},
+		{"ROUTER_IP_V4_V6_MODE", false, ""},
+		{"ROUTER_UNIQUE_ID_HEADER_NAME", false, ""},
+		{"ROUTER_UNIQUE_ID_FORMAT", false, ""},
+		{"ROUTER_H1_CASE_ADJUST", false, ""},
+		{RouterHAProxyThreadsEnvName, true, strconv.Itoa(RouterHAProxyThreadsDefaultValue)},
+		{RouterHTTPIgnoreProbes, false, ""},
+		{"ROUTER_DEFAULT_CLIENT_TIMEOUT", false, ""},
+		{"ROUTER_CLIENT_FIN_TIMEOUT", false, ""},
+		{"ROUTER_DEFAULT_SERVER_TIMEOUT", false, ""},
+		{"ROUTER_DEFAULT_SERVER_FIN_TIMEOUT", false, ""},
+		{"ROUTER_DEFAULT_TUNNEL_TIMEOUT", false, ""},
+		{"ROUTER_INSPECT_DELAY", false, ""},
+		{RouterBackendCheckInterval, false, ""},
```

Comment:
Don't worry about it for this PR, but in the future, please consider using separate commits for refactoring and for functional changes.